zxcvbn-cs/DefaultMatcherFactory.cs

```diff
@@ -19,18 +20,26 @@ class DefaultMatcherFactory : IMatcherFactory
     {
         List<IMatcher> matchers;
 
+        private IEnumerable<string> DictionaryResources()
```

Comment:
Why are these changes in the `DefaultMatcherFactory`? The original library offers a nice `IMatcherFactory` that one can use to implement their own matcher factory. Move these changes into a new file and make sure the library uses our matcher factory instead of `DefaultMatcherFactory`.

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/eee61f5ef53d64b2264891cf37de8b9eb2bf8e15

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/d626983b425765af87bc9127d58cecbb850dcb93